src/view-data/method.ts

```diff
@@ -28,8 +32,13 @@ export interface Method {
   readonly externalDocs: string;
   readonly parameters: TypeSpecParameter[];
   readonly headers: Header[];
+  readonly responseTypes: string;
+
+  /** @deprecated use responseTypes instead, this field will be removed in a future version. */
   readonly successfulResponseType: string;
+  /** @deprecated use responseTypes instead, this field will be removed in a future version. */
   readonly successfulResponseTypeIsRef: boolean;
+  /** @deprecated use responseTypes2 instead */
```

Comment:
responseTypes2 -> responseTypes

Reply:
This JSDoc comment was for a field that no longer exists.

I removed the comment.